src/main/java/com/cglx/Calculator.java

```diff
@@ -37,5 +37,23 @@ public int square(int a) {
 		return result;
 	}
 
+	public int sqroot(int a) {
+
+		return (int) Math.sqrt(a);
+	}
+
+	private void readFile() throws IOException {
+		BufferedReader reader = new BufferedReader(new FileReader("testfile.txt"));
+		StringBuilder stringBuilder = new StringBuilder();
+		char[] buffer = new char[10];
+		while (reader.read(buffer) != -1) {
+			stringBuilder.append(new String(buffer));
+			buffer = new char[10];
+		}
+		reader.close();
+
+		String content = stringBuilder.toString();
+		System.out.println(content);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-01:9990/coding_rules#rule_key=squid%3AS106)